### ISSUE TITLE
ValueError: invalid literal for int() with base 10: '100.0'

### ISSUE THREAD

**Issue by cchall** [radiasoft/sirepo#3770]
`lib.Importer('elegant').parse_file`  fails to import a lattice file previously written by `sirepo.lib.Importer` with an error:

```
/home/vagrant/.pyenv/versions/py3/lib/python3.7/site-packages/sirepo/lib.py:49:_model model=CSRCSBEND field=n_kicks decl=['N KICKS', 'Integer', 4, 'number of kicks. Deprecated. Use N_SLICES.'] value=100.0 exception=invalid literal for int() with base 10: '100.0'
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/tmp/ipykernel_1860/2879720425.py in <module>
      1 # sirepo.lib cannot import just the lattice file so we put it in a dummy command file
      2 # `modify_lattice.ele` is not intended to ever be run
----> 3 mref = lib.Importer('elegant').parse_file('modify_lattice.ele')

~/.pyenv/versions/py3/lib/python3.7/site-packages/sirepo/lib.py in parse_file(self, path)
    109         with pykern.pkio.save_chdir(p.dirpath()):
    110             return SimData(
--> 111                 self.__adapter.parse_file(p),
    112                 p,
    113                 self.__adapter,

~/.pyenv/versions/py3/lib/python3.7/site-packages/sirepo/template/elegant.py in parse_file(self, path)
    120             _verify_files(i, lattice.LatticeUtil.model_name_for_data(i))
    121         r.lattice = l
--> 122         return self._convert(d)
    123 
    124     def write_files(self, data, source_path, dest_dir):

~/.pyenv/versions/py3/lib/python3.7/site-packages/sirepo/lib.py in _convert(self, data)
     58         for x in ('elements', 'commands'):
     59             for m in data.models[x]:
---> 60                 _model(m, LatticeUtil.model_name_for_data(m))
     61         for bl in data.models.beamlines:
     62             if 'positions' in bl:

~/.pyenv/versions/py3/lib/python3.7/site-packages/sirepo/lib.py in _model(model, name)
     45                         model[k] = float(v) if v else 0.
     46                     elif t == 'Integer':
---> 47                         model[k] = int(v) if v else 0
     48             except Exception as e:
     49                 pkdlog('model={} field={} decl={} value={} exception={}', name, k, x, v, e)

ValueError: invalid literal for int() with base 10: '100.0'
```

Lattice file: 
[full_aps_vertical_ramp_colletive.txt](https://github.com/radiasoft/sirepo/files/7017800/full_aps_vertical_ramp_colletive.txt)

**Comment by e-carlin** [comment]
@cchall can you give me the file that was initially fed into sirepo.lib to then generate this file? `n_kicks` should be an integer...

**Comment by cchall** [comment]
Thanks for taking a look @e-carlin. Here is the original lattice.
Changing n_kicks to an int let me proceed with loading the lattice.
It looks like slice_analysis_interval was also changed from from int to float in both the csrcsbend and csrdrift elements, but there was no error for those when I loaded the lattice. 
[original_lattice.txt](https://github.com/radiasoft/sirepo/files/7018109/original_lattice.txt)
